sickbeard/scene_exceptions.py

```diff
@@ -320,7 +320,7 @@ def _xem_exceptions_fetcher():
             if not parsedJSON['data']: 
                 logger.log(u"No data returned from XEM when checking scene exceptions. Update failed for {0}".format
                             (sickbeard.indexerApi(indexer).name), logger.DEBUG)
-                continue
+                break
```

Comment:
I think this part is here, so we can use other indexers. If you break here, and for the first indexer no scene exceptions are returned, it won't look for others. Maybe that's something we want? But it's not in line with the existing functionality.